disnake/state.py

```diff
@@ -875,10 +881,16 @@ def parse_message_reaction_remove(self, data) -> None:
                 if user:
                     self.dispatch("reaction_remove", reaction, user)
 
-    def parse_message_reaction_remove_emoji(self, data) -> None:
+    def parse_message_reaction_remove_emoji(
+        self, data: gateway.MessageReactionRemoveEmojiEvent
+    ) -> None:
         emoji = data["emoji"]
         emoji_id = utils._get_as_snowflake(emoji, "id")
-        emoji = PartialEmoji.with_state(self, id=emoji_id, name=emoji["name"])
+        emoji = PartialEmoji.with_state(
+            self,
+            id=emoji_id,
+            name=emoji["name"],  # type: ignore
```

Comment:
please leave a comment why

Reply:
https://github.com/DisnakeDev/disnake/pull/594/commits/fc2be126a25524eda99b3018e8124c48d2574f31
This should be improved in a future PR, but it's out of scope for this one.